templates/slider.hbs

```diff
@@ -16,7 +16,9 @@
             </div>
             <div class="slider-scaler-wrapper clearfix">
                <div class="slider-scaler component-item-color">
+                  <div class="slider-line component-item-color" style="left: 0;"></div>
                   <div class="slider-markers"></div>
+                  <div class="slider-line component-item-color" style="left: 100%;"></div>
```

Comment:
I'm just curious about the need for these two near identical divs?

Reply:
@ryan-lynch yeah so there is a marker line before and after the indicators to show the beginning and end for the scale. Do you have a better approach in mind?

Reply:
@ryan-lynch the name of the PR is a little misleading; it's not just adding an option to hide the indicator markers, it's also improving the layout for the `_showScale: false` setting, as per [this comment](https://github.com/adaptlearning/adapt_framework/issues/2011#issuecomment-382039272) - so we can now have something like this:
![2018-06-22 16_02_45-adapt version 3 0 demonstration _ question components](https://user-images.githubusercontent.com/1249597/41783813-bc284400-7635-11e8-8bfa-87f0f22bd41d.png)

Looks much better IMHO

Reply:
@nilslp maybe the purpose would be clearer if you gave each of those two divs a more descriptive class name and maybe put the styling into slider.less rather than inline? maybe `.slider-scale-start-marker` and `.slider-scale-end-marker`? probably better for one of the styling experts like @lc-thomasberger or @guywillis to comment on that really